File: php/enqueue.php
<?php

$apiKey = "YOUR_API_KEY";
$endpoint = "https://pendless.atomian.com/api/enqueue/";

$data = [
    "procedure" => "enqueue",
    "url" => "https://example.com/login",
    "prompt" => "Open the login page, enter username and password",
    "title" => "Browser Automation: Login"
];

$options = [
    "http" => [
        "header"  => [
            "Authorization: Bearer $apiKey"
        ],
        "method"  => "POST",
        "content" => json_encode($data)
    ]
];

$context = stream_context_create($options);
$result = file_get_contents($endpoint, false, $context);

$response = json_decode($result, true);
print_r($response);

File: go/enqueue.go
package main

import (
    "bytes"
    "encoding/json"
    "fmt"
    "io/ioutil"
    "net/http"
)

func main() {
    apiKey := "YOUR_API_KEY"
    endpoint := "https://app.pendless.com/api/enqueue/"

    data := map[string]string{
        "procedure": "enqueue",
        "url":       "https://example.com/login",
        "prompt":    "Open the login page, enter username and password",
        "title":     "Browser Automation: Login",
    }

    jsonData, _ := json.Marshal(data)
    req, _ := http.NewRequest("POST", endpoint, bytes.NewBuffer(jsonData))
    req.Header.Set("Authorization", "Bearer "+apiKey)

    client := &http.Client{}
    resp, err := client.Do(req)
    if err != nil {
        panic(err)
    }
    defer resp.Body.Close()

    body, _ := ioutil.ReadAll(resp.Body)
    fmt.Println(string(body))
}
